test/generateTasks.spec.js

```diff
@@ -102,15 +101,18 @@ describe('generateTasks', () => {
   })
 
   it('should not match non-children files', () => {
-    const relPath = path.join(process.cwd(), '..')
-    resolveGitDir.mockReturnValueOnce(relPath)
+    /* eslint-disable */
+    jest.resetModules()
+    jest.setMock('../src/resolveGitDir.js', path.join(process.cwd(), '..'))
```

Comment:
I'm wondering if https://jestjs.io/docs/en/mock-function-api#mockfnmockimplementationoncefn would also work here.

Reply:
As I mentioned, exporting constant looks like a bad design choice since the module does introduces side-effects and it would be better to be explicit about it.